test/unit/ui/app/actions.spec.js

```diff
@@ -1192,40 +1561,49 @@ describe('Actions', () => {
   })
 
   describe('#markPasswordForgotten', () => {
-    let markPasswordForgottenSpy
+    let markPasswordForgottenSpy, forgotPasswordSpy
 
     beforeEach(() => {
-      markPasswordForgottenSpy = sinon.stub(background, 'markPasswordForgotten')
+      markPasswordForgottenSpy = sinon.spy(background, 'markPasswordForgotten')
+      forgotPasswordSpy = sinon.spy(actions, 'forgotPassword')
     })
 
     afterEach(() => {
       markPasswordForgottenSpy.restore()
+      forgotPasswordSpy.restore()
     })
 
     it('calls markPasswordForgotten', () => {
       const store = mockStore()
       store.dispatch(actions.markPasswordForgotten())
+      assert(forgotPasswordSpy.calledOnce)
       assert(markPasswordForgottenSpy.calledOnce)
     })
   })
 
   describe('#unMarkPasswordForgotten', () => {
-    let unMarkPasswordForgottenSpy
+    let unMarkPasswordForgottenSpy, forgotPasswordSpy
 
     beforeEach(() => {
       unMarkPasswordForgottenSpy = sinon.stub(background, 'unMarkPasswordForgotten')
+      forgotPasswordSpy = sinon.spy(actions, 'forgotPassword')
     })
 
     afterEach(() => {
       unMarkPasswordForgottenSpy.restore()
+      forgotPasswordSpy.restore()
     })
 
     it('calls unMarkPasswordForgotten', () => {
       const store = mockStore()
       store.dispatch(actions.unMarkPasswordForgotten())
-      assert(unMarkPasswordForgottenSpy.calledOnce)
+        .then((done) => {
+          assert(forgotPasswordSpy.calledOnce)
+          assert(forgotPasswordSpy.calledWith(false))
+          assert(unMarkPasswordForgottenSpy.calledOnce)
+          done()
```

Comment:
It's unclear what this `done` function is :confused: The `assert` function doesn't even seem to return a promise, so I'm not sure how this is passing.